code/include/z3D/z3D.h

```diff
@@ -10,6 +10,21 @@
 
 // #include "hid.h"
 
+#ifdef Version_EUR
+    #define GAME_ADDR(addr)                                    \
+        ({                                                     \
+            int finalAddr = (addr);                            \
+            if (addr >= 0x41A144 && addr <= 0x43668B) {        \
+                finalAddr += 0x24;                             \
+            } else if (addr >= 0x436690 && addr <= 0x4A5ADF) { \
+                finalAddr += 0x20;                             \
+            }                                                  \
+            finalAddr;                                         \
+        })
```

Comment:
```suggestion
    #define GAME_ADDR(addr)                                       \
        ((addr >= 0x41A144 && addr <= 0x43668B)   ? (addr + 0x24) \
         : (addr >= 0x436690 && addr <= 0x4A5ADF) ? (addr + 0x20) \
                                                  : addr)
```
I believe a macro like this should be computed at compile time, rather than adding a code block that will run each time a base game function is called.
You can achieve this by writing it with ternary operators, which make the compiler realize what you're writing is actually a constant value.